build.gradle

```diff
@@ -47,6 +48,14 @@ pitest {
     threads = 4
 }
 
+gitHooks {
+    hooksDirectory = file('.git/hooks')
+    gradleCommand = './gradlew'
+    hooks = [
+        'pre-push': 'clean check test'
```

Comment:
`pitest` can be added at a later time when we get around to achieving full coverage.